### ISSUE TITLE
free@home does not show all actuator's / accessory's

### ISSUE THREAD

**Issue by Joost1985** [henry-spanka/homebridge-freeathome#34]
I reinstalled my raspberry from scratch this morning with the homebridge image. Then I installed the free@home plugin. Everything seems to work fine (perfect that it is no longer necessary to install the Busch Jaeger API separately). Unfortunately, not all my items are recognized. Could the plugin (incorrectly) put a number of items on the blacklist? I already tried to whitelist everything under 'Mapping' by means of a *, but unfortunately this did not help. Any idea what's going wrong?

I Attached my log.
[log.txt](https://github.com/henry-spanka/homebridge-freeathome/files/5361844/log.txt)

**Comment by Joost1985** [comment]
Okay, I have some extra things sorted out:

I have connected a Philip Hue bridge to my Sysapp. Now it appears that some of the lamps that are linked via the Hue bridge are shown and some are not. So all hard circuits work but unfortunately not all Hue lamps are shown. This is unfortunate because when all lamps are shown, I do not have to install the Hue plugin separately.

**Comment by henry-spanka** [comment]
So to confirm, you are only missing lights from your Hue bridge? Did they work before? Please send me a backup of your System Access point via email and let me know which lights are missing.

**Comment by Joost1985** [comment]
That's right. I have linked several lamps to my Hue bridge. This bridge is linked to my BJ System Access Point. Some of the lamps that are connected to the Hue bridge are shown and some are not. It looks like the lamps of which I can also adjust the heat of the light are not shown and the 'normal' dimmable lights are.

I will send you the backup of my BJ system access point via email.

PS sorry, I'm using google translate ;-)

**Comment by henry-spanka** [comment]
Is this still relevant?
